README.md

```diff
@@ -8,11 +8,17 @@
 #### [Click here](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) to get started!
 
 
-## Emotional Intelligence training
+## Helpful YouTube videos for a career in IT 
 
-1. [Emotional intelligence at work: Why IQ isn't everything](https://youtu.be/7ngIFlmRRPQ)
+1. [A Guide of how to get started in IT in 2023 - Top IT Career paths](https://www.youtube.com/watch?v=XmWkcePhf84)
 
-2. [Ted talk on The Secret to High Performance and Fulfilment](https://youtu.be/HTfYv3IEOqM) 
+2. [What Is Impostor Syndrome And How To Cure It](https://www.youtube.com/watch?v=UdyR6LTsQMc) 
+
+3. [How to learn AWS Cloud in 2023](https://www.youtube.com/watch?v=V-3a2ZG_KMo)
```

Comment:
Nice link! The resources mentioned might be a good future contribution, what do you think?